gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -77,6 +71,12 @@ def construct_argparser(self) -> argparse.ArgumentParser:
             default=None,
             help="Override flag for verifier (choose from among gtsfm/configs/verifier).",
         )
+        parser.add_argument(
+            "--retriever_config_name",
+            type=str,
+            default=None,
```

Comment:
If not specified, what is the default now? Do we have one?

Reply:
It takes it from the config, this is just for the overriding

Reply:
I see, got it, so it's required to be in the config now, like all the other modules now, right?